### ISSUE TITLE
JAVA 7 compatibility

### ISSUE THREAD

**Issue by pas-p** [googleads/googleads-consent-sdk-android#28]
Please add Java 7 compatibility.

**Comment by iordachiv** [comment]
+1
A lot of project is under gradle 3.x and android studio 2.x,  so isn't possible to build with this sdk.

**Comment by kumardexati** [comment]
+1
There is really nothing that requires JAVA8 in the code. This is really annoying.

**Comment by rampara** [comment]
You can update the `build.gradle` file to target Java 7.

```
android {
    ...
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_7
        targetCompatibility JavaVersion.VERSION_1_7
    }
}
...
```

**Comment by rampara** [comment]
Has this issue been resolved on v1.03 of this SDK?

**Comment by kumardexati** [comment]
@rampara Yes 1.0.3 resolved the issue. Thanks for quick response.
